prowler/lib/outputs/json.py

```diff
@@ -63,12 +63,25 @@ def fill_json_asff(finding_output, audit_info, finding, output_options):
             if len(finding.status_extended) > 1000
             else finding.status_extended
         )
+        # Generate resource Tags
+        resource_tags = {}
+        if finding.resource_tags:
+            for tag in finding.resource_tags:
+                if "Key" in tag and "Value" in tag:
+                    resource_tags[tag["Key"]] = tag["Value"]
+                else:
+                    resource_tags.update(tag)
+            if len(resource_tags) == 0:
+                resource_tags = None
+        else:
+            resource_tags = None
```

Comment:
Could you please include this piece of code in a separate function? It'd be great if you could include test for this new function.

Reply:
It should be ready. Let me know if we can improve it

```
❯ poetry run pytest tests/lib/outputs/outputs_test.py::Test_Outputs
============================= test session starts =============================
platform darwin -- Python 3.11.5, pytest-7.4.2, pluggy-1.0.0
Using --randomly-seed=912372200
rootdir: /prowler
configfile: pyproject.toml
plugins: randomly-3.15.0, cov-4.1.0, xdist-3.3.1
collected 26 items

tests/lib/outputs/outputs_test.py ..........................                                                                                                                                                   [100%]

============================= 26 passed in 0.42s ==============================
```

Reply:
Awesome!! 🚀